src/libcore/char.rs

```diff
@@ -8,6 +8,9 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+// XXX todolist: len_utf8, len_utf16 (names), encode_utf8,
+// encode_utf16 (names, API) are still strange.
```

Comment:
Just a note to take another look at this. I've been told that if you s/XXX/TODO/ it'll fail make tidy to make sure you come back to look at it.